README.md

```diff
@@ -244,7 +245,7 @@ TinyExpr parses the following grammar:
     <list>      =    <expr> {"," <expr>}
     <expr>      =    <term> {("+" | "-") <term>}
     <term>      =    <factor> {("*" | "/" | "%") <factor>}
-    <factor>    =    <power> {"^" <power>}
+    <factor>    =    <power> {"*\*" <power>}
```

Comment:
Rather than having `"*\*"` in the markdown, it might make more since to wrap the section in a code block, if that was the reason to use `*\*` instead of `**`.